src/components/MainNavigation/MainNavigation.vue

```diff
@@ -8,8 +8,10 @@
       @click="animateDrawer"
     />
     <ul
+      data-testId="main-navigation-inner"
       :class="[
         'relative',
+        innerClass,
```

Comment:
Nit: can we call this something else? `innerClass` is not the most intuitive name to me. Maybe `ulClass` or `listClass`?